packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -340,38 +346,28 @@ export class Builder extends EventEmitter {
   }
 
   private getProjects() {
-    const projects = this.config.getProjects();
-
-    if (projects) {
-      return Object.values(projects);
-    }
-
-    return [this.config.getProjectConfig()];
+    return getGraphQLProjects(this.config);
   }
 
   private getGlobs() {
     return this.getProjects().reduce<string[]>((globs, project) => {
       return globs.concat(
-        project.includes.map(getFilePath.bind(this, this.config.configDir)),
+        project.includes.map(getGraphQLFilePath.bind(this, this.config)),
       );
     }, []);
   }
 
   private getSchemaPaths() {
-    return this.getProjects().reduce<string[]>((schemas, x) => {
-      if (x.schemaPath) {
-        schemas.push(getFilePath(this.config.configDir, x.schemaPath));
-      }
-      return schemas;
-    }, []);
+    return getGraphQLSchemaPaths(this.config);
   }
 
   private removeDocumentForFile(filePath: string) {
     const project = this.config.getConfigForFile(filePath);
 
     if (project) {
-      const projectName = project.projectName || '';
-      const documents = this.documentMapByProject.get(projectName);
+      const documents = this.documentMapByProject.get(
+        project.resolveProjectName(''),
```

Comment:
This is a bit of a strange one. Normally if you don't provide a default it will use our _sane_ default `projectName` of `GraphQL`. For this tool I didn't want to use a _sane_ default because I care if `projectName` is `falsy` when we are producing the schema types file name.